frontend/src/app/libs/common-components/shared/variables.scss

```diff
@@ -0,0 +1,95 @@
+// grid
+$main-grid-unit: 8px;
+$secondary-grid-unit: 4px;
+//layout
+$layout-outer-margin: 9 * $main-grid-unit;
+$layout-gutter: 3 * $main-grid-unit;
+$layout-columns-quantity: 12;
+$layout-column-width: 10 * $main-grid-unit;
+$layout-total-width: 153 * $main-grid-unit+2 * $layout-outer-margin;
+// colors
+//main
+$color-primary: #367FF6;
+$color-error: #EF5350;
+//additional
+$color-mint: #EF5350;
+$color-purple: #7457E6;
+$color-coral: #FF8A65;
+$color-black: #212121;
+$color-white: #FFFFFF;
+//blacks
+$color-black-100: rgba($color-black);
+$color-black-95: rgba($color-black, .95);
+$color-black-75: rgba($color-black, .75);
+$color-black-50: rgba($color-black, .50);
+$color-black-30: rgba($color-black, .30);
+$color-black-10: rgba($color-black, .10);
+$color-black-5: rgba($color-black, .05);
+$color-black-3: rgba($color-black, .03);
+// whites
+$color-white-100: rgba($color-white);
+$color-white-75: rgba($color-white, .75);
+$color-white-50: rgba($color-white, .50);
+$color-white-30: rgba($color-white, .30);
+$color-white-10: rgba($color-white, .10);
+$color-white-5: rgba($color-white, .05);
+// typography
+$font-main: 'Roboto Bold';
```

Comment:
lets rename it to `font-main-bold` 
also add missing spaces before `'` check all styles

Reply:
done